homeassistant/components/ovo_energy/__init__.py

```diff
@@ -40,6 +40,9 @@ async def async_update_data() -> OVODailyUsage:
         """Fetch data from OVO Energy."""
         now = datetime.utcnow()
         async with async_timeout.timeout(10):
+            await client.authenticate(
```

Comment:
What happens if auth fails?

Reply:
Good point. I'll throw it in a try catch